static/app/views/aiAnalytics/PipelinesTable.tsx

```diff
@@ -83,19 +83,20 @@ export function PipelinesTable() {
     sort = {field: 'time_spent_percentage()', kind: 'desc'};
   }
   const {data, isLoading, meta, pageLinks, error} = useSpanMetrics({
-    search: new MutableSearch('span.op:ai.pipeline.langchain'),
+    search: new MutableSearch('span.category:ai.pipeline'),
```

Comment:
If I'm reading your Relay changes right, this should be `span.category:ai`, since the default categories only take the first part of the `span.op`. , Actually would be even better at `span.module:ai`, for consistency with other modules